tensilelite/Tensile/KernelWriterAssembly.py

```diff
@@ -5284,7 +5284,8 @@ def globalReadGuardKBody(tP):
       loadWidth = tP["globalReadInstruction"].totalWidth
 
       isGlc = True if tP["NonTemporal"]%2==1 else False
-      isSlc = True if tP["NonTemporal"]//2==1 else False
+      isSlc = True if (tP["NonTemporal"]//2)%2==1 else False
```

Comment:
Just a suggestion, better to use bitwise operation. ex:  isSlc = True if tP["NonTemporal"] & 0x2 else False

Reply:
good idea! done